Library/Formula/tutum.rb

```diff
@@ -2,8 +2,8 @@
 
 class Tutum < Formula
   homepage "https://www.tutum.co/"
-  url "https://github.com/tutumcloud/tutum-cli/archive/v0.14.1.tar.gz"
-  sha256 "d908123cc9f761171155f6ccfc66227393fd0dba485963bd40753b5579e0cf71"
+  url "https://pypi.python.org/packages/source/t/tutum/tutum-0.15.0.1.tar.gz"
```

Comment:
The one on `pypi` contains a small patch on the dependency, which doesn't have a release on github